js/util.ts

```diff
@@ -125,3 +125,26 @@ export function deferred(): Deferred {
     reject: reject!
   };
 }
+
+/** Create a IterableIterator. */
+// @internal
+export class CreateIterableIterator<T> implements IterableIterator<T> {
```

Comment:
Can you add a small unit test for this to util_test.py ?

Reply:
@ry  Why not a new `util_test.ts` file?